packages/Box/__tests__/Box.spec.jsx

```diff
@@ -90,11 +90,4 @@ describe('Box', () => {
 
     expect(box).toHaveClassName('a-class')
   })
-
-  it('does not allow custom CSS', () => {
```

Comment:
Should we change this test to allow for `className` instead of removing it?

Reply:
I think it's ok to not have that since that would be testing a React feature rather than something we did. That test was necessary because we actively blocked a React feature from working.